tensorflow-core/tensorflow-core-api/pom.xml

```diff
@@ -150,6 +151,7 @@
             <compilerOption>${project.basedir}/src/main/native/server_jni.cc</compilerOption>
             <compilerOption>${project.basedir}/src/main/native/session_jni.cc</compilerOption>
             <compilerOption>${project.basedir}/src/main/native/tensorflow_jni.cc</compilerOption>
+            <compilerOption>${project.basedir}/src/main/native/tensor_buffers_jni.cc</compilerOption>
```

Comment:
I thought everyone was OK with moving away from writing JNI manually. Why do we need this?

Reply:
Because I need to create an instance of `ByteBuffer` out of a tensor native address and size in case `Unsafe` is not available, and the only way I know of is to do it from JNI. Do you have another way to do it?

Reply:
This is going to do the same as the JNI code in that file:
```java
TF_TensorData(nativeTensor).capacity(TF_TensorByteSize(nativeTensor)).asByteBuffer()
```
Along with a couple of null checks there too if needed.

Reply:
Ok I'll give it a try, thanks

Reply:
Great, that works, bye bye JNI...